src/posit/connect/paginator.py

```diff
@@ -46,4 +45,5 @@ def get_next_page(self) -> List[dict]:
             self.total = response["total"]
         results = response["results"]
         self.seen += len(results)
+        self.page_number += 1
```

Comment:
Was this a bug you uncovered using these? The change here seems very sensible, but I'm surprised it was the other way around before.

Reply:
Yes, see https://github.com/posit-dev/posit-sdk-py/pull/86/commits/c2b20d8f87a62850140e779dbe7ead6061764ea3. Users.find_one was never incrementing the page number, but the previous test only checked finding something on the first page. After factoring out the test fixtures, I could more easily set up additional tests to confirm different features of this, and encountered this.